src/std/io/exception.d

```diff
@@ -12,7 +12,15 @@ class IOException : Exception
         this.msg = msg.move;
     }
 
-    override void toString(scope void delegate(const scope char[]) @safe sink) const
+    override void toString(scope void delegate(in char[]) sink) const
+    {
+        // Wrapping into @trusted is not strictly correct because the sink might escape
+        // the slice (allthough it's highly unlikely)
+        // Retain the old behavior and implement a transition later
+        return toString((const scope char[] line) @trusted => sink(line));
+    }
+
+    void toString(scope void delegate(const scope char[]) @safe sink) const @trusted
```

Comment:
This seems to be triggering a failure in older dmd (Looks like we use 2.093), it seems like it thinks this should be an override. Any ideas?

Reply:
Seems to happen still on latest compiler (at least with a simple test), are you sure this is valid?